src/commands/bigQueryCostEstimate.ts

```diff
@@ -14,12 +14,14 @@ export class BigQueryCostEstimate {
     private telemetry: TelemetryService,
   ) {}
 
-  async estimateCost() {
+  async estimateCost({ returnResult }: { returnResult?: boolean }) {
```

Comment:
Why do we need to have returnResult, can't we not simply ignore it if we don't need it?

Reply:
@mdesmet This change was needed in bigquery cost estimator in insights panel to show the response.

Reply:
I repeat my question, why not always return the result and get rid of the parameter?